docs/spec/lightclient/detection/detection.md

```diff
@@ -435,41 +828,47 @@ func ForkDetector(ls LightStore)  {
 			auxLS.Init
 			auxLS.Update(LightStore.LatestTrusted(), StateVerified);
 			auxLS.Update(sh,StateUnverified);
-			result := VerifyToTarget(s, auxLS, sh.Header.Height)
-			if result = (_,ResultSuccess) || (_,EXPIRED) {
+			LS,result := VerifyToTarget(secondary, auxLS, sh.Header.Height)
+			if (result = ResultSuccess || result = EXPIRED) {
```

Comment:
In case result = EXPIRED that means that secondary is not responsive. But as this does not allow us to verify if conflicting header is a fork, we shouldn't report it. Note that this is not a problem as the important scenario is faulty-primary, correct secondary, in which case we will not have timeout.

Reply:
Actually, `EXPIRED` referred to some older version of verification.md. The Error code does not exist anymore, and we should remove it and just check for `ResultSuccess`